origin-tests/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "origin-tests",
+  "version": "1.0.0",
+  "description": "Integration tests for Origin Protocol",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "mocha --timeout 10000 --exit test"
+  },
+  "author": "Tom Linton <tom@originprotocol.com>",
+  "license": "ISC",
```

Comment:
Tiny thing but I think we're MIT for everything

Reply:
Thanks, think this was the default for `npm init`, fixed it up!